Json-Rpc/Handler.cs

```diff
@@ -185,12 +186,6 @@ public void UnRegister(string key)
         /// <returns></returns>
         public JsonResponse Handle(JsonRequest Rpc, Object RpcContext = null, Action<JsonResponse> callback = null)
         {
```

Comment:
I'm a little confused about the changes I'm seeing here. Was
`Action<JsonResponse> callback = null`
from one of your prior commits? 
If so, any chance you could squash your commits so the code is easier to review?
https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/